sdk/keyvault/azure_security_keyvault_certificates/tests/certificate_client.rs

```diff
@@ -53,12 +52,10 @@ async fn certificate_roundtrip(ctx: TestContext) -> Result<()> {
         certificate_policy: Some(DEFAULT_POLICY.clone()),
         ..Default::default()
     };
-    let operation = client
-        .create_certificate("certificate-roundtrip", body.try_into()?, None)
-        .await?
-        .into_body()
+    client
+        .create_certificate("certificate-roundtrip", body.try_into()?, None)?
+        .wait()
         .await?;
```

Comment:
@RickWinter, @ronniegeraghty, @LarryOsterman, @analogrelay here's an example of usage, similar to `tokio::process::Command::wait().await`.

Reply:
Talked in today's team meeting and people are okay with this.